### PR TITLE
fix(scss) box-shadows and adding elevate utilities

### PR DESCRIPTION
**Pull request by sanlucca**

### CODE REVIEW

**Comment by LuccaIntegration** [comment]
jenkins auto deploy http://lucca-front.lucca.local/PR-1141

**Review by RipripKipkip — APPROVED** [review]

File: packages/scss/src/theming/_commons.scss [hunk 1]
```diff
@@ -41,20 +41,20 @@ $commons: (
 	),
 	elevations: (
 		elevation-1: unquote("0px 1px 2px rgba(0, 0, 0, 0.06), 0px 2px 8px rgba(0, 0, 0, 0.04)"),
-		elevation-2: unquote("0px 1px 2px rgba(0, 0, 0, 0.02), 0px 2px 6px rgba(0, 0, 0, 0.05)"),
-		elevation-3: unquote("0px 2px 4px rgba(0, 0, 0, 0.02), 0px 4px 10px rgba(0, 0, 0, 0.06)"),
-		elevation-4: unquote("0px 3px 6px rgba(0, 0, 0, 0.04), 0px 4px 16px rgba(0, 0, 0, 0.08)"),
-		elevation-5: unquote("0px 4px 8px rgba(0, 0, 0, 0.06), 0px 12px 32px rgba(0, 0, 0, 0.08)"),
-		elevation-6: unquote("-2px 2px 8px rgba(0, 0, 0, 0.04), -12px 6px 24px rgba(0, 0, 0, 0.06)"),
+		elevation-2: unquote("0px 0px 0px 1px rgba(0, 0, 0, 0.03), 0px 1px 2px rgba(0, 0, 0, 0.02), 0px 2px 6px rgba(0, 0, 0, 0.05)"),
```
Comment (vvalentin-lucca) [inline packages/scss/src/theming/_commons.scss:44]: 0 without units to please the linter?

**Comment by LuccaIntegration** [comment]
jenkins auto deploy http://lucca-front.lucca.local/PR-1141

**Review by vvalentin-lucca — APPROVED** [review]

**Comment by LuccaIntegration** [comment]
jenkins auto deploy http://lucca-front.lucca.local/PR-1141

**Pull request merged**